tbtc-ts/src/sweep.ts

```diff
@@ -0,0 +1,161 @@
+// @ts-ignore
+import bcoin from "bcoin"
+// @ts-ignore
+import wif from "wif"
+import { BigNumber } from "ethers"
+import {
+  RawTransaction,
+  UnspentTransactionOutput,
+  Client as BitcoinClient,
+  isCompressedPublicKey,
+} from "./bitcoin"
+import {
+  createDepositScript,
+  getActiveWalletPublicKey,
+  DepositData,
+} from "./deposit"
+
+// TODO: Add description
+export async function createScriptSig(
```

Comment:
I think signing should be part of this method as well since `createScriptSig` is responsible for creating the entire content of the `scriptSig`/`witness` field. I'd also rename `createScriptSig` to, for example, `resolveDepositScript` and return the "artifact" (`scriptSig` or `witness`) able to unlock the given deposit P2SH or P2WSH output. I think all differences between P2SH and P2WSH should be also handled here so we should also set `transaction.inputs[i].script` or `transaction.inputs[i].witness` here.

Reply:
Done in https://github.com/keep-network/tbtc-v2/pull/96/commits/9784c3bdd63a99b0565c89b2a9d1b2ce635f11a0.